api/lib/DBClient.js

```diff
@@ -7,6 +7,22 @@ function DBClient() {
     this._client = Cloudant({account:user, password:pw});
 }
 
+DBClient.prototype.get = function(db, from, size) {
+    if ( typeof from !== "number") from = 0;
+    if ( typeof size !== "number") size = null;
+    return new Promise((res, rej) => {
+        let database = this._client.db.use(db);
+        database.list({ include_docs: true }, function(err, body) {
```

Comment:
you can define "skip" and "limit" query strings in the first argument of `database.list`, so you don't have to manually filter your results after querying, eg.:

```js
database.list({
    include_docs: true,
    skip: from,
    limit: size
})
```

Make sure that `size` is either a number or 'undefined', 'undefined' means 'no-limit'.

By the way, this is also more efficient than retrieving the entire DB and then filtering what we want.

Reply:
Nice! I did not know about this two parameters :laughing: 
I'm gonna implement that!